src/modules/packages/packages.conf

```diff
@@ -106,7 +106,7 @@ pacman:
 #       useful if you have a static package archive on the install media.
 #       The *pacman* package manager is the only one to specially support
 #       this operation (all others treat this the same as *install*).
-# - *remove*, *try_remove*: will call the package manager to
+# - *remove*, *remove_deps*, *try_remove*, *try_remove_deps*: will call the package manager to
```

Comment:
This needs a better explanation of the differences.

Reply:
In your opinion, should I separate them in a new line providing a dedicated description or all of them in the current existing line editing the description?